extras/signers.json

```diff
@@ -45,13 +45,15 @@
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
   "ozDefender": {
-    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7"
+    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
+    "checkpointer_v2": "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F"
```

Comment:
```suggestion
    "relayerV1": "0xfb7d0D001BC8D0bC998071C762BfF53EE31b725F"
```

Reply:
Checkpointer is the wrong nomenclature here and is an artifact of this being added to our addressbook before we had all the context?  What do you think about doing it something like this to not propagate such foolishness?

Reply:
there are lots of labels that could be finetuned in this way, but not sure of the (breaking) implications though